gtkfi/ArcSDM
# Implement Rasterize for FeatureLayer label parameter in MLP functions

# Description

This PR implements the `rasterize_vector` function in the `rasterize.py` file to improve its functionality and handle NoData values more effectively. Now, only raster layers or feature layers are allowed as parameters for X and y in MLP functions to ensure that it works properly. Also, early stopping has been set to true for both MLP functions.

# Testing
1. use data that has the exact same amount of columns for example RC_AS and RC_La.
2. For labels use IOCG_Deps_Prosp_Occs from EIS.
3. Make sure that the MLP function starts training the data.
